src/interface/storage.thrift

```diff
@@ -457,11 +458,12 @@ struct LookUpEdgeIndexResp {
 }
 
 struct LookUpIndexRequest {
```

Comment:
If we modify the thrift protocol structure, maybe cause incompatible ?

Reply:
> If we modify the thrift protocol structure, I’m afraid it maybe cause incompatible ?

No, go ahead. let me resolve this conflict later.